### PR TITLE
Update runtime changelog of 20240313.v1 and 20240319.v1

### PR DESCRIPTION
**Pull request by thy09**
# Description

This pull request includes a significant update to the `runtime-change-log.md` file in the `docs/cloud/azureai` directory. The changes include the addition of two new runtime image versions, `20240319.v1` and `20240313.v1`, with their respective new features and bug fixes.

* [`docs/cloud/azureai/runtime-change-log.md`](diffhunk://#diff-4afb18084f9779997e9a3b1d2584b90355aa45d3d78620e5a236fe2a1b19b28cR18-R31): Added two new runtime image versions. The `20240319.v1` version introduces a new feature that adds a "seed" parameter to the LLM tool chat API. The `20240313.v1` version fixed several bugs, including a bug that caused a failure when calling a flow with a flow function, an issue that marked the error type as a user error if a run was archived before processing by the runtime, and an error that occurred when the start async flow received a non-200 status.
# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by PeiwenGaoMS — DISMISSED** [review]

File: docs/cloud/azureai/runtime-change-log.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ You can check the runtime image version from the flow execution log:
 ## Change log
 Default runtime image is continuously updated, and here we record the new features and fixed bugs of each image version.
 
+### 20240319.v1
+
+#### New features
+
+- Add param "seed" to LLM tool chat api
```
Comment (Bowen-Guo) [inline docs/cloud/azureai/runtime-change-log.md:22]: Add a period in the end.

File: docs/cloud/azureai/runtime-change-log.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ You can check the runtime image version from the flow execution log:
...
+
+### 20240313.v1
+
+#### Bugs fixed
+
+- Fix the bug that call flow with a flow function will fail.
+- Mark the error type as user error If a run is archived before processing by runtime.
```
Comment (Bowen-Guo) [inline docs/cloud/azureai/runtime-change-log.md:29]: "before runtime" is fine.

File: docs/cloud/azureai/runtime-change-log.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ You can check the runtime image version from the flow execution log:
...
+- Return error response when start async flow got not 200 status.
```
Comment (Bowen-Guo) [inline docs/cloud/azureai/runtime-change-log.md:30]: "not 200" does not seem grammar-wise correct.

File: docs/cloud/azureai/runtime-change-log.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ You can check the runtime image version from the flow execution log:
...
+
+#### New features
+
+- Add param "seed" to LLM tool chat api
+
+### 20240313.v1
+
+#### Bugs fixed
+
+- Fix the bug that call flow with a flow function will fail.
```
Comment (Bowen-Guo) [inline docs/cloud/azureai/runtime-change-log.md:28]: Seem to me this sentence is not grammarly-correct.

File: docs/cloud/azureai/runtime-change-log.md (hunk 2 above)
Comment (thy09) [reply docs/cloud/azureai/runtime-change-log.md]: Updated

File: docs/cloud/azureai/runtime-change-log.md (hunk 4 above)
Comment (thy09) [reply docs/cloud/azureai/runtime-change-log.md]: Updated

**Review by Bowen-Guo — APPROVED** [review]

**Review by PeiwenGaoMS — APPROVED** [review]

**Pull request merged**